src/pcapi/core/users/api.py

```diff
@@ -888,3 +890,21 @@ def fraud_manager(user: User, phone_number: str) -> typing.Generator:
     except exceptions.PhoneValidationAttemptsLimitReached as error:
         fraud_api.handle_phone_validation_attempts_limit_reached(user, error.attempts)
         raise
+
+
+def update_last_connection_date(user):
+    previous_connection_date = user.lastConnectionDate
+    last_connection_date = datetime.now()
```

Comment:
```suggestion
    last_connection_date = datetime.utcnow()
```

cf. mon autre commentaire